### PR TITLE
Apply autogenerated new API reference

### PR DESCRIPTION
**Pull request by hsorellana**
This PR adds the new objects with its corresponding description to the official reference

### CODE REVIEW

**Comment by jonfoust** [comment]
Resolved along with the fix for the missing postcss module #201

**Comment by hsorellana** [comment]
Hey @jonfoust why this PR is closed and not merged?? Because i was adding the documentation for Backfill API

**Comment by aLekSer** [comment]
@jonfoust This is not fixed, there is no `backfill` on https://open-match.dev/site/docs/reference/api/ page.

**Comment by jonfoust** [comment]
Sorry I may have had tab in a window for PRs to close for once the fix went in mistakenly

File: site/content/en/docs/Reference/api.md [hunk 1]
```diff
@@ -689,9 +689,9 @@ messages are not finalized and still subject to possible change or removal.
 | ----- | ---- | ----- | ----------- |
 | id | [string](#string) |  | Id represents an auto-generated Id issued by Open Match. |
 | search_fields | [SearchFields](#openmatch.SearchFields) |  | Search fields are the fields which Open Match is aware of, and can be used when specifying filters. |
-| extensions | [Backfill.ExtensionsEntry](#openmatch.Backfill.ExtensionsEntry) | repeated | Customized information not inspected by Open Match, to be used by the match making function, evaluator, and components making calls to Open Match. Optional, depending on the requirements of the connected systems. |
+| extensions | [Backfill.ExtensionsEntry](#openmatch.Backfill.ExtensionsEntry) | repeated | Customized information not inspected by Open Match, to be used by the MatchMakingFunction, evaluator, and components making calls to Open Match. Optional, depending on the requirements of the connected systems. |
```
Comment (jonfoust) [inline site/content/en/docs/Reference/api.md:692]: To be a bit more in tune with the rest of the docs, if I may suggest, probably using "Match Function" here. Open to keeping it the same.

**Review by jonfoust — CHANGES_REQUESTED** [review]

File: site/content/en/docs/Reference/api.md (hunk 1 above)
Comment (hsorellana) [reply site/content/en/docs/Reference/api.md:692]: you mean `Match Making Function`, right??

Comment (jonfoust) [reply site/content/en/docs/Reference/api.md]: ```MatchMakingFunction``` -> ```MatchFunction```
Yeah that is what I was referring to. I assume you're speaking about Match Function service in the same context you referred to the evaluator so I figured this change would be a bit more consistent.

Comment (hsorellana) [reply site/content/en/docs/Reference/api.md]: oh, i see. Ok, no problem, will change it then

**Review by jonfoust — APPROVED** [review]
LGTM

**Pull request merged**